### PR TITLE
draft plan

### PR DESCRIPTION
**Pull request by annefou**
- Starting a new CodeRefinery lesson for the CodeRefinery Instructor training (I was not entirely sure how to create a new CodeRefinery lesson; let me know if it is not correct) and drafted a plan. 

Feedback welcome.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-fork_me_ribbon_url: https://github.com/coderefinery/example-lesson
-title: "Example lesson"
+fork_me_ribbon_url: https://github.com/coderefinery/instructor-training
+title: "CodeRefinery for contributors and instructors"
 start_time: 540  # 9:00
```
Comment (bast) [inline _config.yml]: The start_time looks outdated - in more recent lessons we have abandoned start time and rather indicate episode lengths. please have a look at lessons like git-intro.

Comment (annefou) [reply _config.yml]: Thanks! 
So I will also update https://github.com/coderefinery/example-lesson as I used it for the template.

Comment (bast) [reply _config.yml]: Yes please. I forgot we had that repo :-)

**Comment by wikfeldt** [comment]
yes, great starting point. I'll merge

**Pull request merged**